provider/cloudflare.go

```diff
@@ -158,10 +183,7 @@ func (p *CloudFlareProvider) Zones(ctx context.Context) ([]cloudflare.Zone, erro
 
 		for _, zone := range zonesResponse.Result {
 			if !p.domainFilter.Match(zone.Name) {
-				continue
-			}
-
-			if !p.zoneIDFilter.Match(zone.ID) {
```

Comment:
Just to understand it a bit better, this is dropped because it's not working?

Reply:
Dropped because if there was any zoneIDFilter then the above branch would be taken and this code is unreachable.